### ISSUE TITLE
Want to add feature to Beam_2.ipynb to calculate deflection

### ISSUE THREAD

**Issue by Ewdy** [Ewdy/Structual-Design#7]
At present, Beam_2.ipynb calculates the shear and moment for simply supported beam with partially distributed load. It does not yet calculate deflection. I believe that I can easily implement this using Moment-Area Theorems where the change in slope between any two points is equal to the area of the M/EI diagram between these two points. See Structural Analysis by R.C. Hibbeler 4th Edition, pages 280 to 288.

**Comment by Ewdy** [comment]
I think it would be interesting to compare calculation times and error for deflection derived per above with deflection calculated by converting the uniform load into a number of point loads and deriving deflection based on the sum of these elemental loads.

**Comment by Ewdy** [comment]
I have added a feature to calculate deflection by converting a partially distributed uniform load into several equivalent point loads. This works well with sufficiently large sample sizes. However, it can likely be made considerably faster. I am closing this issue because the deflection feature has been added. I may open another issue in future if and when I want to calculate deflection using the Moment-Area theorems as originally intended.
